storage/src/test/java/org/apache/kafka/tiered/storage/utils/TieredStorageTestUtils.java

```diff
@@ -53,7 +54,6 @@
 import static org.apache.kafka.server.log.remote.storage.RemoteLogManagerConfig.REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP;
 import static org.apache.kafka.server.log.remote.storage.RemoteLogManagerConfig.REMOTE_STORAGE_MANAGER_CLASS_NAME_PROP;
 import static org.apache.kafka.server.log.remote.storage.RemoteLogManagerConfig.REMOTE_STORAGE_MANAGER_CONFIG_PREFIX_PROP;
-import static org.apache.kafka.storage.internals.log.CleanerConfig.LOG_CLEANER_ENABLE_PROP;
```

Comment:
It seems we don't need this change, right?

Reply:
If we use static import, it will mark deprecated warning at import, thus I think we should not use static import
![CleanShot 2025-04-17 at 00 43 51](https://github.com/user-attachments/assets/93c80154-7b68-447e-a510-5d78b1758979)